tests/main/preseed-reset/task.yaml

```diff
@@ -0,0 +1,66 @@
+summary: Check that preseeded chroot can be re-set.
+description: |
+  This test checks that preseeding of Ubuntu cloud images with snap-preseed
+  can be undone with --reset flag.
+
+systems: [ubuntu-19.10-*,ubuntu-20.04-*]
+
+environment:
+  IMAGE_MOUNTPOINT: /mnt/cloudimg
+
+prepare: |
+  # the get_image_url_for_nested_vm is a convenient helper that returns
+  # a cloud image url matching current $SPREAD_SYSTEM.
+  #shellcheck source=tests/lib/nested.sh
+  . "$TESTSLIB/nested.sh"
+  wget "$(get_image_url_for_nested_vm)" -O cloudimg.img
+  mkdir -p "$IMAGE_MOUNTPOINT"
+
+restore: |
+  rm -f before-preseeding.txt
+  rm -f after-reset.txt
+
+  # any of the restore commands can fail depending on where execute part stopped,
+  # account for that with ||true.
+  umount_ubuntu_image "$IMAGE_MOUNTPOINT" || true
+
+execute: |
+  find_files() {
+    find "$IMAGE_MOUNTPOINT/etc/" "$IMAGE_MOUNTPOINT/var/"
```

Comment:
are we sure we don't touch anything in usr?

Reply:
Thanks for being paranoid! Indeed, we create symlinks for bash completion... I'll address this.

Reply:
Done.